wapitiCore/main/wapiti.py

```diff
@@ -123,7 +123,7 @@ def __init__(self, root_url, scope="folder", session_dir=None, config_dir=None):
         self.forms = []
         self.attacks = []
 
-        self.color = 0
+        self.color = False
```

Comment:
Is it a boolean that represents whether colors are supported by the terminal? If so, would it be possible to rename the variable to something more explicit like `self.color_enabled` or something?